io.liri.BaseApp.yaml

```diff
@@ -0,0 +1,52 @@
+id: io.liri.BaseApp
+branch: stable
+runtime: org.kde.Platform
+sdk: org.kde.Sdk
+runtime-version: 5.11
+separate-locales: false
+modules:
+- name: qbs
+  cleanup-platform:
+  - "*"
+  sources:
+  - type: archive
+    url: http://download.qt.io/official_releases/qbs/1.11.0/qbs-src-1.11.0.tar.gz
```

Comment:
Have you spoken with the KDE runtime folks about getting `qbs` in there? I don't know the status of it but that would be ideal in the long term.

Reply:
Nope I haven't, but we can ask @aleixpol about it :)

Reply:
And yes including `qbs` in the KDE runtime would indeed be better for other reasons: for instance I would like to make a QtCreator flatpak and I would like to have access to `qbs` which means it must either be in the KDE SDK or as an extension.

Yesterday I tried to make a `qbs` extension of `org.kde.Sdk` but the build fails, probably because of a missing entry in `add-extensions` in the KDE manifest.

Reply:
It would be fine to have qbs in org.kde.Sdk, sure.